### ISSUE TITLE
训练时卡住了

### ISSUE THREAD

**Issue by BrianXiao1** [zhangbo2008/wav2lip384_my2#2]
很好整合工作！只是在训练时遇到了以下问题：

use_cuda: True
BatchSize: 64
Loaded data train: <torch.utils.data.dataloader.DataLoader object at 0x7f23ff8e4fa0>
Loaded data test: <torch.utils.data.dataloader.DataLoader object at 0x7f249aca7100>
Learning rate:  0.0001
total trainable params 65054464
Training From Scratch !!!
Starting Epoch: 0

训练时一直卡在这里没动过，我 ubuntu18.04 cuda11.6 torch=1.12.0
